### PR TITLE
Added __init__ files and updated files for pylint

### PR DESCRIPTION
**Pull request by colin99d**
Added __init__ files, updated .coveragerc, and made fx_view work with pylint.

### CODE REVIEW

File: gamestonk_terminal/forex/fx_view.py [hunk 1]
```diff
@@ -10,13 +10,28 @@
...
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.orders as orders
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.instruments as instruments
+
+# pylint: disable=R0402
 import oandapyV20.endpoints.positions as positions
+
```
Comment (jmaslek) [inline gamestonk_terminal/forex/fx_view.py:27]: can this not just be `from oandapyV20.endpoints import (pricing,accounts,...)`

Comment (colin99d) [reply gamestonk_terminal/forex/fx_view.py:27]: I was just getting my pylint to work. I didn't add this file.

**Comment by jmaslek** [comment]
Was fx_view not passing before?

**Comment by colin99d** [comment]
> Was fx_view not passing before?

**Comment by colin99d** [comment]
I am not sure. I did not edit the view except to make it work with my pylint.

**Comment by jmaslek** [comment]
Im just confused.  If it wasn't working with pylint then every other commit would fail

**Comment by DidierRLopes** [comment]
Could have been due to the init in the forex folder which may trigger they pylint to check all files in that dir since that becomes a module

File: gamestonk_terminal/forex/fx_view.py (hunk 1 above)
Comment (DidierRLopes) [reply gamestonk_terminal/forex/fx_view.py:27]: Can you do @jmaslek suggestion, so on the top do

```
from oandapyV20.endpoints import (
   pricing,
   accounts,
   orders,
   ...
)
```
and then you don't need the `pylint: disable=R0402` anymore

**Comment by colin99d** [comment]
Ok I switched to the new method. This way is much cleaner but I still needed an ignore because `Imports from package oandapyV20 are not grouped (ungrouped-imports)`

**Comment by DidierRLopes** [comment]
> Ok I switched to the new method. This way is much cleaner but I still needed an ignore because `Imports from package oandapyV20 are not grouped (ungrouped-imports)`

So, the reason for that is because when you import oanda related stuff the import commands are separate by other imports.

image

You just need to put all these commands next to each other

**Comment by DidierRLopes** [comment]
Ideally, we should use `pylint: disable` only when we can't do it in another way. Since pylint is there to enforce good coding development

**Comment by colin99d** [comment]
Cool. I did not not it was that easy to fix. Thanks for that!

**Comment by DidierRLopes** [comment]
> Cool. I did not not it was that easy to fix. Thanks for that!

Np, let's merge this once tests pass!! Thanks!!

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**